piker/ui/_ohlc.py

```diff
@@ -421,3 +399,28 @@ def boundingRect(self):
             )
 
         )
+
+    def paint(
```

Comment:
This was just a method move to put `.paint()` at the end of the class def like the OHLC curve type.